src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/GetRecallNotificationComponentTest.kt

```diff
@@ -135,18 +137,30 @@ class GetRecallNotificationComponentTest : ComponentTestBase() {
 
     assertThat(recallDoc.content, equalTo(expectedBase64Pdf))
 
-    val firstRecallNotification =
-      documentRepository.findLatestVersionedDocumentByRecallIdAndCategory(recall.recallId, RECALL_NOTIFICATION)!!
-    assertThat(firstRecallNotification.version, equalTo(1))
-    val firstRevocationOrder =
+    val latestRevocationOrder =
       documentRepository.findLatestVersionedDocumentByRecallIdAndCategory(recall.recallId, REVOCATION_ORDER)!!
-    assertThat(firstRevocationOrder.version, equalTo(1))
+    assertThat(latestRevocationOrder.version, equalTo(1))
+    assertThat(latestRevocationOrder.fileName, equalTo(FileName("BADGER NATALIA B1234 REVOCATION ORDER.pdf")))
+  }
 
-    // Reset mocks to reassure that revocation order is not regenerated
-    gotenbergMockServer.resetAll()
-    gotenbergMockServer.stubGenerateRecallSummary(expectedPdf, "RECALL NOTIFICATION")
-    gotenbergMockServer.stubGenerateLetterToProbation(expectedPdf, "IF AT ANY TIME THE PROBATION SERVICE RECEIVES INFORMATION THAT MAY AFFECT THE VALIDITY OF THE RECALL ACTION")
-    gotenbergMockServer.stubGenerateOffenderNotification(expectedPdf)
+  @Test
+  fun `get recall notification for STANDARD in custody recall includes correct letter to probation`() {
+    val userId = authenticatedClient.userId
+    val recall = authenticatedClient.bookRecall(
+      BookRecallRequest(
+        nomsNumber,
+        FirstName(firstName),
+        null,
+        LastName("Badger"),
+        CroNumber("1234/56A"),
+        LocalDate.now()
+      )
+    )
+    updateRecallWithRequiredInformationForTheRecallNotification(recall.recallId, userId, true, STANDARD)
+
+    gotenbergMockServer.stubGenerateRevocationOrder(expectedPdf, firstName)
```

Comment:
I find these tests a bit unconvincing when they re-use `expectedPdf` for multiple responses and also the value derived from that, `expectedBase64Pdf` in e.g. `assertThat(recallDoc.content, equalTo(expectedBase64Pdf))`?
Did we improve on that in some other similar tests already?

Reply:
Not that I know of.